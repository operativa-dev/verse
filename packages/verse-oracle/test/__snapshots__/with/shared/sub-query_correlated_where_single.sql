-- Executing SQL: Parameters: []
select t1.ArtistId, t1.Name
from Artist t1
where t1.ArtistId = (
   select t3.ArtistId
   from Album t3
   where (t1.ArtistId = t3.ArtistId) and (t1.Name = 'Buddy Guy')
   fetch next 2 rows only
)

