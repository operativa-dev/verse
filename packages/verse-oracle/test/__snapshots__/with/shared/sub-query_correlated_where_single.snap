[
  Artist {
    "albums": [],
    "artistId": 15,
    "name": "Buddy Guy",
  },
]